docs/reference/migration/migrate_3_0.asciidoc

```diff
@@ -285,6 +285,12 @@ The two individual setters for existence() and nullValue() were removed in favou
 optional constructor settings in order to better capture and validate their interdependent
 settings at construction time.
 
+==== NotQueryBuilder
+
+The NotQueryBuilder which was deprecated in 2.1.0 is removed. As a replacement use BoolQueryBuilder
+with added mustNot() clause. So instead of using `NotQueryBuilder(filter)` now use 
+`BoolQueryBuilder().mustNot(filter)`. 
```

Comment:
sorry for being picky, can you prepend the snippets this `new`? They are supposed to call constructors no? That's it ;)